### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jukoson**
Add clearer instructions to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-
+Swarm64 DA Docker Compose will assist to start instance of S64 DA or native PSQL.
+Once the instance is up and running you can connect through a psql client or run
+a benchmark using the [Swarm64 TPC Toolkit](https://github.com/swarm64/tpc-toolkit).
 
 # Prerequisites
 
 - Python 3.6 or higher
-- docker-compose
+- docker-compose (eg. use: python3.6 -m pip install docker-compose)
```
Comment (sdressler) [inline README.md]: That command is not necessarily available everywhere, could also be just `python3` IMO.

File: README.md [hunk 2]
```diff
@@ -23,8 +24,15 @@ This docker-compose is to start an instance of S64 DA or PSQL or both.
...
+   of data and 384GB of system RAM.
+
 3. To start a database instance, call `docker-compose` accordingly:
 
-   - S64 DA on Intel PAC: `docker-compose -f docker-compose-s64da-pac up`
-   - S64 DA on Xilinx Alveo `docker-compose -f docker-compose-s64da-xilinx up`
-   - Native PostgreSQL: `docker-compomse -f docker-compose-psql up`
+   - S64 DA on Intel PAC Arria10: `docker-compose -f docker-compose-s64da-pac.yml up`
+   - S64 DA on Xilinx Alveo U200: `docker-compose -f docker-compose-s64da-xilinx.yml up`
+   - Native PostgreSQL: `docker-compomse -f docker-compose-psql.yml up`
```
Comment (sdressler) [inline README.md]: Typo: "compose"

**Review by sdressler — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jukoson) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (jukoson) [reply README.md]: Added python3.6 / python3

File: README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ a benchmark using the [Swarm64 TPC Toolkit](https://github.com/swarm64/tpc-toolk
 # Prerequisites
 
 - Python 3.6 or higher
-- docker-compose (eg. use: python3.6 -m pip install docker-compose)
+- docker-compose (eg. use: python3.6 / python3 -m pip install docker-compose)
```
Comment (sdressler) [inline README.md]: Hm, not happy with that, think it is more confusing now than before. How about just removing the `python3.6 / ` part?

**Review by sdressler — CHANGES_REQUESTED** [review]

**Review by sdressler — APPROVED** [review]

**Pull request merged**